shogun-config/src/main/java/de/terrestris/shogun/config/DefaultWebSecurityConfig.java

```diff
@@ -43,7 +43,17 @@ default void customHttpConfiguration(HttpSecurity http) throws Exception {
                     "/v3/api-docs",
                     "/v3/api-docs/swagger-config",
                     // Enable anonymous access to GraphiQL
-                    "/graphiql/**"
+                    "/graphiql/**",
+                    // Enable anonymous access to entity endpoints
+                    // will be secured via permission evaluators
+                    "/applications",
+                    "/applications/**",
+                    "/layers",
+                    "/layers/**",
+                    "/files/**",
+                    "/files",
+                    "/imagefiles/**",
+                    "/imagefiles"
```

Comment:
Do we really need to allow non GET requests to anonymous users?

Reply:
I didn't know that it is possible to specifiy this by the method. I guess this would also makes sense for some of the other (mostly static resource) pathes here.